components/cluster/command/reload.go

```diff
@@ -150,7 +150,8 @@ func buildReloadTask(
 				Data:   dataDirs,
 				Log:    logDir,
 				Cache:  spec.ClusterPath(clusterName, spec.TempConfigPath),
-			}).Build()
+			}).
+			BuildAsStep(fmt.Sprintf("  - Refresh config %s -> %s", inst.ComponentName(), inst.GetHost()))
```

Comment:
```suggestion
			BuildAsStep(fmt.Sprintf("  - Refresh config %s -> %s", inst.ComponentName(), inst.ID()))
```
The same host may contain multiple instances.